### DISCUSSION TITLE
Apollo Vite Version 5.0 - Dim Color Scheme Layout Setting Not Working

### DISCUSSION THREAD

**Discussion by everbach** [primefaces/community#785] [PrimeVue Templates] [answered]
I just purchased and installed the Apollo Vue 5.0 Vite theme and noticed the colorScheme: 'dim' setting does not work when editing the layout.js file. 

When I update this setting to 'dim' and load the theme, 'Dim' is selected in the layout selector (cog icon) on the right of screen, but the Light theme is still showing. (Other settings like menuMode (static, slim etc.) seem to be working fine.) Example:

const layoutConfig = reactive({
    menuMode: 'static', // static, slim, and slim-plus are confirmed to work
    menuTheme: 'colorScheme',
    colorScheme: 'dim', //light works; dim and dark change the ui selector but not the actual color
    theme: 'indigo',
    scale: 14
});

The same thing happens when trying to update the App.vue file via this method: 

<script setup>
import { useLayout } from '@/layout/composables/layout';

const { layoutConfig } = useLayout();
layoutConfig.colorScheme.value = 'dim';
</script>

Is this a known bug and do you have a recommended fix or timeline for theme update? 

Thanks in advance for your help!

**Comment by tugcekucukoglu** [comment, marked as answer]
It seems the default theme is still light theme. To change the default theme, you could change this line to 'theme-dim'

`index.html`

```
<link id="theme-link" href="/layout/styles/theme/theme-dim/indigo/theme.css" rel="stylesheet"/>
```

**Reply by everbach** [reply]
That worked - thanks so much for your help!

**Reply by raviue** [reply]
Hi There,

I am looking for documentation to upgrade Apollo Vite Version 4.0 to 5.0

Could you help me how to do it?

Thanks in advance for the help!
